docs/trace/getting-started-aspnetcore/README.md

```diff
@@ -0,0 +1,88 @@
+# Getting Started with OpenTelemetry .NET in 5 Minutes - ASP.NET Core Application
+
+First, download and install the [.NET
+SDK](https://dotnet.microsoft.com/download) on your computer.
+
+Create a new web application:
+
+```sh
+dotnet new webapp -o aspnetcoreapp
+cd aspnetcoreapp
+```
+
+Install the
+[OpenTelemetry.Exporter.Console](../../../src/OpenTelemetry.Exporter.Console/README.md)
+packages:
+
+```sh
+dotnet add package OpenTelemetry.Exporter.Console --prerelease
+dotnet add package OpenTelemetry.Extensions.Hosting --prerelease
+dotnet add package OpenTelemetry.Instrumentation.AspNetCore --prerelease
```

Comment:
I am neutral here... The getting started could be strictly "how to manually create a span in 5 minutes". A "next step" could show the instrumentation library stuff, instead of manual span creation.

Showing instrumentation library here will truly make it a 5 min tutorial and is the likely path a normal user would take.

Will discuss in SIG meetings to get more views on this.

Reply:
> Showing instrumentation library here will truly make it a 5 min tutorial and is the likely path a normal user would take.

Sort of what I was thinking. Get up and running quickly, see some data for incoming requests. Manually creating spans could be a next topic/advanced doc for users who are interested 🤷‍♀️

Reply:
> Manually creating spans could be a next topic/advanced doc

+1. Yes, I've always thought of manually generating telemetry as an advanced topic not a step #1 topic. A lot of folk are coming to OpenTelemetry having used other proprietary products many of which just require a simple install/configuration to see some magic.